util/topsql/topsql.go

```diff
@@ -167,8 +167,6 @@ func MockHighCPULoad(sql string, sqlPrefixs []string, load int64) bool {
 		if time.Since(start) > 12*time.Millisecond*time.Duration(load) {
 			break
 		}
-		for i := 0; i < 10e5; i++ {
-		}
```

Comment:
I don't think this can be removed.

Reply:
I think it is an empty loop to wait for something. so why don't they use ```sched_yield```？

Reply:
@crazycs520 PTAL

Reply:
Sorry, it is my problem. it have been fixed.